kafka/include/userver/kafka/consumer_scope.hpp

```diff
@@ -114,6 +115,19 @@ class ConsumerScope final {
     /// has stopped and leaved the group
     void AsyncCommit();
 
+    /// @brief Retrieves the minimum and maximum offsets for the specified Kafka topic and partition.
+    /// @warning This is a blocking call
+    /// @param topic The name of the Kafka topic.
+    /// @param partition The partition number of the Kafka topic.
+    /// @return A struct with minimum and maximum offsets for the given topic and partition.
+    OffsetRange GetOffsetRange(const std::string& topic, std::int32_t partition) const;
```

Comment:
also std::uint32_t for partition

Reply:
try to use std::string_view instead of const std::string

Reply:
string_view does not have c_str, but data() doesn't garanted null terminated string